lisa_commands/commands.py

```diff
@@ -31,101 +29,169 @@ def parse_mention(message_text):
     # the first group contains the username, the second group contains the remaining message
     return (matches.group(1), matches.group(2).strip().lower()) if matches else (None, None)
 
-def parse_command(message_text):
+def commands():
+    """
+    returns the list of the commands, their description and the function they
+    map to
     """
-    Finds the command and the query from the string
-    :message_text the text we're searching
-    :returns command mentioned and query string
+
+    return {
+        'anime me': {
+            'function': anime_me,
+            "description": "use text after command to query anime news network for anime info"
+        },
+        'call the cops': {
+            'function': call_the_cops,
+            "description": "responds with an image of anime cops with the caption You Called?"
+        },
+        'decide': {
+            'function': decide,
+            "description": "picks one of the words at radom in the text after command"
+        },
+        'flip coin': {
+            'function': flip_coin,
+            "description": "use text after command to query giphy for gifs"
+        },
+        'gif me': {
+            'function': gif_me,
+            "description": "use text after command to query giphy for gifs"
+        },
+        'help': {
+            'function': help_command,
+            "description": "send user a direct message to user with list of commands"
+        },
+        'image me': {
+            'function': image_me,
+            "description": 'use text after command to search for an image (use suffix "from [google|bing]" if you want to specify a search engine)'
+        },
+        'kill me': {
+            'function': kill_me,
+            "description": ' responds with a gif with kill me as a search'
+        },
+        'manga me': {
+            'function': manga_me,
+            "description": "use text after command to query anime news network for manga info"
+        },
+        'put it back': {
+            'function': put_it_back,
+            "description": "when you are done flipping the table and it's time to clean up"
+        },
+        'reverse me': {
+            'function': reverse_me,
+            "description": "reverses text after command"
+        },
+        'shame': {
+            'function': shame,
+            "description": 'shame the text after command'
```

Comment:
nit `shame the user after command`

Reply:
technically it shames whatever you enter